### ISSUE TITLE
`listenTo` in `Marty.createContainer` doesn't support deeply-nested object properties

### ISSUE THREAD

**Issue by moretti** [martyjs/marty-lib#7]
So, in Marty 0.10 you can register entities using an object literal:

``` javascript
import { Application } from 'marty';
import UserStore from './stores/userStore';

class MyApp extends Application {
    constructor(options) {
        this.register({
            user: {
                store: UserStore,
            },
        });
    }
}
```

And in theory you should be able to listen for a nested store by doing something like:

``` javascript
Marty.createContainer(User, {
    listenTo: [
        'user.store',
    ],
   // ...
});
```

I tried the example above, but I got an error from `storeObserver`: 

```
Could not find the store user.store
```

https://github.com/martyjs/marty-lib/blob/master/src/core/storeObserver.js#L71

Perhaps you need to use something like [`selectn`](https://github.com/wilmoore/selectn) to resolve deeply-nested object properties?

Thanks! :smile:

**Comment by meafmira** [comment]
+1

**Comment by adamschoenemann** [comment]
+1

**Comment by bigardone** [comment]
+1
I posted my issue here also:

https://github.com/martyjs/marty/issues/332

**Comment by jhollingworth** [comment]
Deployed in v0.10.2

**Comment by moretti** [comment]
Thank you!
I didn't know that lodash had a one-liner like `_.get()`, that's pretty cool! :smile:

**Comment by adamschoenemann** [comment]
I don't think marty-lib was rebuilt for 0.10.2! None of the changes are visible in modules.

**Comment by bigardone** [comment]
Yes, same happened to me... I updated Marty to v0.10.2 but those changes are not included in the script.

**Comment by jhollingworth** [comment]
Just published v0.10.3 which actually includes this fix. I've created
https://github.com/martyjs/marty-lib/issues/15 to ensure these mistakes
don't happen again

On 3 June 2015 at 17:29, Ricardo García notifications@github.com wrote:

> Yes, same happened to me... I updated Marty to v0.10.2 but those changes
> are not included in the script.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/martyjs/marty-lib/issues/7#issuecomment-108511761.

## 

http://twitter.com/jhollingworth | http://github.com/jhollingworth |
http://linkedin.com/in/jhollingworth | +44 7866 843 864
